verigreen-collector-impl/src/main/resources/configurations/jbossCacheConfig.xml

```diff
@@ -27,7 +27,7 @@ See the License for the specific language governing permissions and limitations
 		<loader class="org.jboss.cache.loader.FileCacheLoader" async="false"
 			fetchPersistentState="true">
 			<properties>
-				location=/${env.VG_HOME}/cacheloader
+				location=cacheloader
```

Comment:
The change worked for me(infact it is used to be this way before somebody made this change to put the cache loader in VG_HOME).  The problem is that it will put the cacheloader in the working folder(instead of in VG_HOME) where the executable is launched. Note however, even without my change there is a bug. the ${env.VG_HOME}  already has a front slash at the start.  So providing one more front slash at the start, leads to error. In my case though the logic just didn't interpret the env.VG_HOME variable, it was trying to look for `/${env.VG_HOME}/cacheloader` folder literally. Not sure what was missing.